api/users/serializers.py

```diff
@@ -11,20 +36,28 @@ class UserSerializer(JSONAPISerializer):
         'family_name',
         'id'
     ])
+    parser_classes = (JSONParser,)
```

Comment:
I'm not sure. There were some new things that @billyhunt had to do to get this working because there are new interactions with DRF that we hadn't been doing. Certainly worth trying it without.